homeassistant/data_entry_flow.py

```diff
@@ -216,9 +200,11 @@ def __init__(
         """Initialize the flow manager."""
         self.hass = hass
         self._preview: set[str] = set()
-        self._progress: dict[str, FlowHandler] = {}
-        self._handler_progress_index: dict[str, set[FlowHandler]] = {}
-        self._init_data_process_index: dict[type, set[FlowHandler]] = {}
+        self._progress: dict[str, BaseFlowHandler] = {}
+        self._handler_progress_index: dict[str, set[BaseFlowHandler]] = {}
+        self._init_data_process_index: dict[type, set[BaseFlowHandler]] = {}
+
+    _flow_result: Callable[..., _FlowResultT]
```

Comment:
```suggestion
    _flow_result: type[_FlowResultT]
```
I would also suggest to move that above the `__init__` method like with all other type declarations in classes.